docs/attributes-registry/db.md

```diff
@@ -59,61 +59,62 @@ If a parameter has no name and instead is referenced only by index, then `<key>`
+| `elasticsearch`      | Elasticsearch                                      | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `filemaker`          | FileMaker                                          | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `firebird`           | Firebird                                           | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `firstsql`           | Deprecated, no replacement at this time.           | ![Deprecated](https://img.shields.io/badge/-deprecated-red)<br>Removed.                          |
+| `geode`              | Apache Geode                                       | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `h2`                 | H2                                                 | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `hanadb`             | SAP HANA                                           | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `hbase`              | Apache HBase                                       | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `hive`               | Apache Hive                                        | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `hsqldb`             | HyperSQL DataBase                                  | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `influxdb`           | InfluxDB                                           | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `informix`           | Informix                                           | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `ingres`             | Ingres                                             | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `instantdb`          | InstantDB                                          | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `interbase`          | InterBase                                          | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `intersystems_cache` | InterSystems Caché                                 | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
```

Comment:
_100% off-topic comment: wow, it's really called `Caché`_

Reply:
right?! there is just one caché in the world!